test/pytest/test_metrics_kf.py

```diff
@@ -297,18 +318,18 @@ def test_metrics_location_var_snapshot_enabled_rdonly_dir():
     test_utils.stop_torchserve()
     test_utils.delete_all_snapshots()
     # First remove existing logs otherwise it may be a false positive case
-    for f in glob.glob('logs/*.log'):
+    for f in glob.glob("logs/*.log"):
         os.remove(f)
-    RDONLY_DIR = path.join(test_utils.ROOT_DIR,'rdonly_dir')
-    os.environ['METRICS_LOCATION'] = RDONLY_DIR
+    RDONLY_DIR = path.join(test_utils.ROOT_DIR, "rdonly_dir")
+    os.environ["METRICS_LOCATION"] = RDONLY_DIR
     try:
         run_metrics_location_var(custom_path=RDONLY_DIR, no_config_snapshots=False)
-        requests.post('http://127.0.0.1:8081/models?url=mnist.mar')
-        assert len(glob.glob('logs/access_log.log')) == 1
-        assert len(glob.glob('logs/model_log.log')) == 1
-        assert len(glob.glob('logs/ts_log.log')) == 1
-        assert len(glob.glob('logs/config/*snap*.cfg')) == 1
-        assert len(glob.glob(RDONLY_DIR + '/logs/model_metrics.log')) == 0
-        assert len(glob.glob(RDONLY_DIR + '/logs/ts_metrics.log')) == 0
+        requests.post("http://127.0.0.1:8081/models?url=mnist.mar")
+        assert len(glob.glob("logs/access_log.log")) == 1
+        assert len(glob.glob("logs/model_log.log")) == 1
+        assert len(glob.glob("logs/ts_log.log")) == 1
+        assert len(glob.glob("logs/config/*snap*.cfg")) == 1
+        assert len(glob.glob(RDONLY_DIR + "/logs/model_metrics.log")) == 0
+        assert len(glob.glob(RDONLY_DIR + "/logs/ts_metrics.log")) == 0
     finally:
-        del os.environ['METRICS_LOCATION']
\ No newline at end of file
+        del os.environ["METRICS_LOCATION"]
```

Comment:
I can't find the important code that was changed in the metrics file because of the linting changes as well. What should I pay most attention to

Reply:
The function `load_module_from_py_file` was added in both test_metrics & test_metrics_kf.py